nextroute/check/check.go

```diff
@@ -0,0 +1,28 @@
+package check
+
+import (
+	"context"
+
+	"github.com/nextmv-io/sdk/connect"
+	"github.com/nextmv-io/sdk/nextroute"
+)
+
+// ModelCheck is the check of a model returning a [Output].
+func ModelCheck(
```

Comment:
```suggestion
func Model(
```

I would consider renaming this. When you are using it elsewhere, it will be `check.ModelCheck` which has a stutter. Instead, `check.Model` is just as clear.

Reply:
Model is not really a verb... and if it would be it does not communicate what the method does. API should communicate what it does (not the package name). Unfortunately Go has no function/method overloading so we can not use the API Check which would have been appropriate. Also we can not use CheckModel as Linter does not allow the method to start with the package name....